### ISSUE TITLE
Decimal default value not parsed correctly

### ISSUE THREAD

**Issue by dchem** [mafintosh/protocol-buffers-schema#22]
```
message tester { 
  optional float number1 = 1 [default = 1.79769];
  optional string string1 = 2;
}
```
Encoding and decoding `{ string1: "" }` yields:

```
{ number1: 1, string1: '' }
```

**Comment by mafintosh** [comment]
fixed in latest protocol-buffers 3.1.4

**Comment by dchem** [comment]
:100: Thank you!
